src/logging.cpp

```diff
@@ -71,6 +72,9 @@ bool BCLog::Logger::StartLogging()
 
     // dump buffered messages from before we opened the log
     m_buffering = false;
+    if (m_buffer_lines_skipped > 0) {
```

Comment:
In commit "logging: Limit early logging buffer" (3d4b1f4af406276671b5d0115574dea6b4a64340)

Since the log limit discards old log prints instead of skipping new prints, I think it would be a little clearer if "skipped" was replaced with "discarded" throughout this commit.

Reply:
Changed